### PR TITLE
Replace telegra.ph with te.legra.ph

### LINKED ISSUE #315: Alternative for telegra.ph
Telegra.ph banned in india, could you add graph.org

### PR DESCRIPTION
**Pull request by SpEcHiDe**
Fixes #315

### CODE REVIEW

**Comment by Jigarvarma2005** [comment]
Create telegraph client once when start use many times, possible na?

File: plugins/misc/telegraph/__main__.py [hunk 1]
```diff
@@ -69,7 +71,25 @@ async def telegraph_(message: Message):
...
+                    config.LOG_CHANNEL_ID,
+                    TE_LEGRA_PH_ACCESS_TOKEN
+                )
+            ).text
+        else:
+            at = await pool.run_in_thread(create_access_token)(header)
+            await message.client.send_message(
+                config.LOG_CHANNEL_ID,
+                at
+            )
```
Comment (Harsh-br0) [inline plugins/misc/telegraph/__main__.py:87]: This seems spammy cuz it will send a token on each restart of server

Comment (SpEcHiDe) [reply plugins/misc/telegraph/__main__.py:87]: after receiving once, we can update env var 😅

Comment (SpEcHiDe) [reply plugins/misc/telegraph/__main__.py:87]: i guess better solution would be someone experienced with mongoDB could PR .

**Pull request closed without merging**